.circleci/config.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       LANG: C.UTF-8
       FASTLANE_ITUNES_TRANSPORTER_PATH: .bundle
```

Comment:
If you really install iTMS, this path should be updated to the actual installation path of iTMS.